### ISSUE TITLE
cumulative PV generation years PV lifetime instead of modelling period

### ISSUE THREAD

**Issue by palomaortegaa** [CLOVER-energy/CLOVER#158]
### Describe the bug
Cumulative PV generation output was for 20 years instead of modelling period

### Expected behaviour
Cumulative PV generation should use modelling period instead of PV lifetime

### Desktop
**(please complete the following information)**
 - OS: all
 - Installation setup: downloaded source from Github
 - Version: 5.0.7b2
 - Branch, if relevant: master

**Comment by hamishbeath** [comment]
Does the model use that metric to calculate the LCUE figures?

**Comment by palomaortegaa** [comment]
@hamishbeath , no, it's just for the PV generation metric where it uses the data downloaded from Renewables Ninja. So, everything else is fine, just this one 😛
